client/src/api/ipfsUploader.ts

```diff
@@ -0,0 +1,45 @@
+import IpfsClient from 'ipfs-http-client';
+
+/**
+ * Promisified version of FileReader.readAsArrayBuffer 
+ */
+const readBlobAsArrayBuffer = (blob: Blob): Promise<ArrayBuffer> => (
+  new Promise((resolve, reject) => {
+    const reader = new FileReader();
+    
+    reader.onload = () => { resolve(reader.result as ArrayBuffer); };
+    reader.onerror = () => { reject(reader.error); reader.abort(); };
+
+    reader.readAsArrayBuffer(blob);
+  })
+)
+
+const ipfsClient = IpfsClient('http://localhost:5001')
```

Comment:
Hardcoded URL instead of reading it from the config

Reply:
Config will come in the next PR. I am still figuring out how to use it properly.